src/frontend/src/components/Authentication/helpers/authentication.ts

```diff
@@ -68,117 +61,361 @@ const getEmployeeIdFromEmail = async (email: string): Promise<string> => {
+    }
+  };
+
+  return useMutation(login, {
+    onSuccess,
+    onError,
   });
-  return COGNITO_CLIENT.send(loginCommand);
 };
 
-export const initiatePasswordReset = async (email: string): Promise<void> => {
-  try {
-    const forgotPasswordCommand = new ForgotPasswordCommand({
-      ClientId: COGNITO_CONFIG.clientId,
-      Username: email,
-    });
 
-    await COGNITO_CLIENT.send(forgotPasswordCommand);
-    return Promise.resolve();
-  } catch (err) {
-    return Promise.reject(err);
+
+interface PasswordParams {
+  email: string;
+  onSuccess: (data: any) => void;
+  onError: (err: unknown) => void;
+}
+
+export const initiatePasswordReset = ({
+  email,
+  onSuccess,
+  onError,
+}: PasswordParams) => {
+  const reset = async (): Promise<any> => {
+    const response = await fetch(getAuthApiUrlForResetPassword(email), {
+      method: 'GET', 
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    switch (response.status) {
+      case 200:
+        return Promise.resolve(200);
+      default:
+        return Promise.reject(new Error("Unknown error occurred"));
+    }
   }
+
+  return useMutation(reset, {
+    onSuccess,
+    onError,
+  });
 };
 
-export const confirmPasswordReset = async (
-  email: string,
-  newPassword: string,
-  verificationCode: string
-) => {
-  try {
-    const confirmPasswordResetCommand = new ConfirmForgotPasswordCommand({
-      ClientId: COGNITO_CONFIG.clientId,
-      Username: email,
-      ConfirmationCode: verificationCode,
-      Password: newPassword,
-    });
 
-    await COGNITO_CLIENT.send(confirmPasswordResetCommand);
-    return Promise.resolve();
-  } catch (err) {
-    return Promise.reject(err);
-  }
+// export const initiatePasswordReset = async (email: string): Promise<void> => {
+//   try {
+//     const forgotPasswordCommand = new ForgotPasswordCommand({
+//       ClientId: COGNITO_CONFIG.clientId,
+//       Username: email,
+//     });
+
+//     await COGNITO_CLIENT.send(forgotPasswordCommand);
+//     return Promise.resolve();
+//   } catch (err) {
+//     return Promise.reject(err);
+//   }
+// };
+
+
+
+interface ConfirmPasswordResetParams {
+  email: string;
+  verificationCode: string;
+  newPassword: string;
+  idToken: string;
+  onSuccess: (data: any) => void; 
+  onError: (err: unknown) => void;
+}
+
+export const confirmPasswordReset = ({
+  email,
+  verificationCode,
+  newPassword,
+  idToken,
+  onSuccess,
+  onError,
+}: ConfirmPasswordResetParams) => {
+    const confirm = async (): Promise<any> => {
+      console.log(verificationCode)
+      const response = await fetch(API_URLS.PASSWORD, { 
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${idToken}`,
+        },
+        body: JSON.stringify({ email, code: verificationCode, password: newPassword }),
+      });
+      switch (response.status) {
```

Comment:
After updating the backend to include `400` responses for user errors, please handle the `400` responses to display a useful error message to the user so they know what to do next.

See this as an example:
https://github.com/jlui17/W2FHR/blob/cec83627d2271ade708318d3feae382f31a8a906/src/frontend/src/components/Authentication/helpers/authentication.ts#L280-L302